en/docs/guides/Provisioning/outbound-provisioning-with-google.md

```diff
@@ -0,0 +1,148 @@
+# Configure Outbound Provisioning with Google
+
+This topic provides instructions on how to configure Google as the
+Identity Provider to provision users from WSO2 Identity Server. The
+service provider in this scenario is WSO2 Identity Server(WSO2 IS). 
+
+!!! tip "Before you begin!"
+    -   You need to have a Google domain. Click
+        [here](https://www.bettercloud.com/monitor/the-academy/create-google-apps-domain-three-easy-steps/)
+        for more information on creating the domain.
+    
```

Comment:
Add a separator line between subheadings to make the page neater 

syntax: --------

Reply:
please apply wherever necessary

Reply:
Done